store/sqlstore/user_store.go

```diff
@@ -59,7 +60,20 @@ func newSqlUserStore(sqlStore *SqlStore, metrics einterfaces.MetricsInterface) s
 	return us
 }
 
+func (us SqlUserStore) validateAutoResponderMessageSize(notifyProps model.StringMap) error {
+	if notifyProps != nil {
+		if msg := notifyProps[model.AutoResponderMessageNotifyProp]; utf8.RuneCountInString(msg) > us.Post().GetMaxPostSize() {
+			return errors.New("Auto responder message size can't be more than the allowed Post size")
```

Comment:
It would be helpful to log the auto responder message size and the post size here.

Reply:
Done